packages/terra-form-field/src/Field.scss

```diff
@@ -13,10 +13,13 @@
   }
 
   .label-group {
+    font-family: var(--terra-form-field-label-font-family, 'Roboto');
```

Comment:
The default theme should be the open-source theme and looks like the font-family for source-source is HelveticaNeue.

Reply:
updated. Thanks.